src/app.css

```
@@ -276,3 +279,115 @@
     transform: rotate(360deg);
   }
 }
+
+@theme inline {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)